.github/scripts/release-verification.sh

```diff
@@ -25,6 +25,14 @@ if [[ -z "${RELEASE_VERSION_NUMBER}" ]]; then
     exit 1
 fi
 
+OS=linux
+ARCH=arm64
+GITHUB_ORG=radius-project
+GITHUB_REPO=radius
+RADIUS_CLI_ARTIFACT="rad_${OS}_${ARCH}"
+DOWNLOAD_BASE="https://github.com/${GITHUB_ORG}/${GITHUB_REPO}/releases/download"
```

Comment:
xxx/releases/download seems to be unreachable. should be only xxxx/releases

Reply:
I copied from our install.sh script, I think releases/download is the right path when suffixed with the right filename: https://github.com/radius-project/radius/blob/796fc5d0758c526526d2cead3e8f0d976ab7f828/deploy/install.sh#L116C10-L116C10

`curl https://github.com/radius-project/radius/releases/download/v0.26.8/rad_darwin_arm64 -sSL -o rad` seems to work for me